docs/release-notes/v13.0.0.md

```diff
@@ -5,14 +5,10 @@ sidebar_label: v13.0.0
 image: "img/release-note-link-preview.png"
 ---
 
-_May 09th, 2024_
+_May 30th, 2024_
 
-:::info
-Mia-Platform Console v13.0.0 is **now in Preview** and will be generally available on May 30th.
-
-Console SaaS users can try out v13.0.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
-
-For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::caution
+This version contains a bug in the Flow Manager Configurator that prevents saving when moving from lower Console versions if the Flow Manager design section is not opened. This bug has been solved in `v13.0.1`. For on-premise installations we recommend not to use this version. Please check the `v13.0.1` Release notes [here](/release-notes/v13.0.1.md).
```

Comment:
```suggestion
This version is affected by a bug in the Flow Manager Configurator that prevents saving the configuration if this section is not opened during the session in the Design Area before saving. This bug occurs in case the Flow Manager Configurator has been already adopted in lower Console versions. This bug has been solved in `v13.0.1`. For on-premise installations we recommend not to use this version. Please check the `v13.0.1` Release notes [here](/release-notes/v13.0.1.md).
```